tests/unit/backend/corpora/dataset_processing/test_process.py

```diff
@@ -590,3 +591,55 @@ def test__cxg_not_created_when_metadata_extraction_fails(
         dataset = Dataset.get(self.session, dataset_id)
         processing_status = dataset.processing_status
         self.assertEqual(None, processing_status.cxg_status)
+
+    @patch("backend.corpora.dataset_processing.process.make_seurat")
```

Comment:
Why so many mocks? 😄 

None of these functions matter to the outcome of the behavior being tested in this unit test. 

e.g. whether `process_cxg` runs or not is irrelevant to Seurat conversion. Likewise, for these 2 unit tests, we don't care about writing to the database (`update_db`). In fact, writing to a real database is undesirable in a unit test since it makes them run much slower.

The fact that so many mocks are required to test something this simple is probably an indication that the code needs to be refactored. But that will be left as an exercise for another day.

Reply:
Much simpler 😄 

Fixed